packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -375,30 +380,48 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
    * @param rawError
    */
   generateClientEventErrorPayload(rawError: any) {
+    if (rawError.name) {
+      if (isBrowserMediaErrorName(rawError.name)) {
+        return this.getErrorPayloadForClientErrorCode({
+          serviceErrorCode: undefined,
+          clientErrorCode: BROWSER_MEDIA_ERROR_NAME_TO_CLIENT_ERROR_CODES_MAP[rawError.name],
```

Comment:
the mapping of `rawError.name` to `clientErrorCode` is not 1 tot 1 it's N to 1, so we're losing some information here. It would be good to still send the `rawError.name` to CA, not sure if there is some field we can use for that?

Reply:
that is true, Ill add it to the code